tracer/src/Datadog.AutoInstrumentation.NativeLoader/util.h

```diff
@@ -37,6 +38,30 @@ static fs::path GetCurrentModuleFolderPath()
     return {};
 }
 
+static ::shared::WSTRING GetDatadogLogsDirectoryPath()
```

Comment:
this is a duplicate of what we have already.
You could extract the `logs folder computation` from `log path computation` and reuse it
(see in `shared\src\native-src\pal.h`)
This can be done in another review

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a